fintech-examples/fintech-ui/src/app/redirect-after-consent-denied/redirect-after-consent-denied.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="card card-frame">
+  <div class="d-flex justify-content-center mt-5">
+    <img alt="redirection" class="card-img" src="assets/icons/icons8-network 2.png" />
+  </div>
+  <div class="card-body">
+    <h4 class="card-title">Consent denied</h4>
+    <p class="card-text text-center">Your consent was denied! <br>
```

Comment:
Why do you use a countdown solution instead of a button?
What if the user was distracted by something?

Reply:
The countdown is there set for 5 seconds: long enough for user to read the message and there is a button to go back to bank search if the user does not want to wait.

We could increase the countdown to 10 seconds for example or remove for good if needed